acquia/cli
# CLI-644: Add an ACLI Dockerfile

**Motivation**

Some users are struggling to install ACLI because of dependencies or lack of privileged access to install software. Having a Dockerfile is a way to have a consistent experience across various operating systems.

Another use case is customers might want to start from the base ACLI image and add additional business logic on top of it.

**Proposed changes**
Add a `Dockerfile` at the root of the ACLI repo.

**Alternatives considered**
N/A

**Testing steps**
* Clone the repo
* Make sure Docker is running
* From within the root of the repo, build the image:

```
❯ docker build -t acli .
```

* Run an ACLI command and get the container to exit immediately. E.g.

```
❯ docker run -it acli php /usr/local/bin/acli --version
Acquia CLI 1.21.1
```

* Get into the container and run interactive commands:

```
❯ docker run -it acli sh
/ # acli --version
Acquia CLI 1.21.1
```

Note: to get the checksum, run:

```
❯ sha256sum acli.phar
ec6368888922d4d036917b033261920423352c4c40d702c00a316a6553263e7c  acli.phar
```

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/master/CONTRIBUTING.md#building-and-testing) to set up your development environment.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. (add specific steps for this pr)

**Merge requirements**
